driver/xrt/src/accl.cpp

```diff
@@ -80,10 +80,8 @@ ACCL::~ACCL() {
   delete cclo;
 }
 
-void ACCL::deinit() {
-  debug("Removing CCLO object at " + debug_hex(cclo->get_base_addr()));
-
-  cclo->printDebug();
+void ACCL::reset() {
```

Comment:
out of alignment with the surrounding code

Reply:
The indentation is consistent